kortex_gazebo/launch/spawn_kortex_robot.launch

```diff
@@ -156,7 +158,7 @@
         </node>
 
         <!-- Test if homing the robot with MoveIt ended correctly -->
-        <test test-name="paramtest_gazebo_initialization" pkg="rostest" type="paramtest">
+        <test if="$(arg enable_test)" test-name="paramtest_gazebo_initialization" pkg="rostest" type="paramtest">
```

Comment:
Since using [rostest](http://wiki.ros.org/rostest) instead of `roslaunch` does exactly this, I wonder why this parameter is useful?

Reply:
correct, removed